static/sass/_lagunita.scss

```diff
@@ -287,6 +287,11 @@ $video-thumb-url: '../themes/lagunita/images/stanford-online-video-thumb.png';
     }
 }
 
+.course-info header.course-profile {
```

Comment:
Which upstream SASS selector is overriding this?

Can we try making this selector more explicit,
as to avoid needing the `!important` bits below?

```sass
div#content section.course-info header.course-profile {}
```